app/src/main/java/dev/andrylat/app/CreditCardHandler.java

```diff
@@ -0,0 +1,116 @@
+package dev.andrylat.app;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class CreditCardHandler {
+    private StringBuilder errors = new StringBuilder();
+    private StringBuilder message = new StringBuilder();
+    private StringBuilder brand = new StringBuilder();
+    private CreditCardBrandDeterminer brandDeterminer = new CreditCardBrandDeterminer();
+    private static final int VALID_CREDIT_CARD_NUMBER_LENGTH = 16;
+    
+    public String check(String input) {
+    	validate(input);
+    	
+    	if (errors.length() > 0) {
+    		message.append("Card number is invalid.\n")
```

Comment:
Why did you do in this way? 
For HTTP web service (Task 1.2) what will you do with that string? For HTTP you should create JSON with array of errors. it means that your validator should return LIST of errors. And "user" of that validator should use that list as he prefers: concatenate, make message, create JSON, etc.